Sources/Rendering/WebXR/RenderWindowHelper/index.js

```diff
@@ -267,6 +357,8 @@ function vtkWebXRRenderWindowHelper(publicAPI, model) {
 
 const DEFAULT_VALUES = {
   initialized: false,
+  drawControllersRay: false,
+  inputSourceToRay: {},
```

Comment:
if you have a dict as a value, you can't use `DEFAULT_VALUE` but you need to use the  `defaultValue()` approach